src/util.js

```diff
@@ -15,6 +15,8 @@ var Util = {
 
     rWidgetParts: /^\[\[\u2603 (([a-z-]+) ([0-9]+))\]\]$/,
     rWidgetRule:  /^\[\[\u2603 (([a-z-]+) ([0-9]+))\]\]/,
+    rWidgets: /\[\[\u2603 (([a-z-]+) ([0-9]+))\]\]/g,
+    rWidgetSplit: new RegExp("(\\[\\[\u2603 [a-z-]+ [0-9]+\\]\\])", "g"),
```

Comment:
can this use a regex literal? regex literals are generally a bit easier to read (you don't need to double escape things in literals, for example), and can sometimes be faster (since they can be compiled at parse time)